flybywiresim/a32nx
# fix(efb): Upgrade @flybywiresim/react-components to version 0.3.1

Fixes #5011

## Summary of Changes
The bug is described in issue #5011. Previously, when using the distance measuring tool. The units of distance would not update on tooltips of a path that was not finished yet. Changing units would only affect existing paths. Now, all tooltips should update even if the path is not finished yet.

I have made a change to the plugin we are using for the map and published the change via the react-component (https://github.com/flybywiresim/react-components/pull/11) library, which I am now updating here (and on the website).

## Screenshots (if necessary)



Before (notice all different units):
![](https://user-images.githubusercontent.com/5309174/120883089-d0823200-c60d-11eb-8d7e-b60dd6ba61c2.png)

After:
![Screenshot 2021-06-26 124900](https://user-images.githubusercontent.com/22713769/123510732-6c3b1700-d67d-11eb-8ca3-d477c8911602.png)


## References





## Additional context



Discord username (if different from GitHub): BlueberryKing#6641

## Testing instructions

1. Load up anywhere
2. Open EFB.
3. Use the distance measuring tool to draw a path. Finish the path by clicking on the endpoint.
4. Draw a new path, do not finish it. Instead, just use the mouse (which should still be moving the endpoint of the path) to change the units on the control panel.
5. Observe how all tooltips have the units selected on the control panel.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
